### ISSUE TITLE
addEventListener third param type

### ISSUE THREAD

**Issue by mkusher** [Microsoft/TypeScript#14188]
According to https://dom.spec.whatwg.org/#interface-eventtarget `addEventListener`'s third param should be of type `boolean | AddEventListenerOptions`. Should it be changed in ts libs and if so, then where should it be changed?
https://github.com/Microsoft/TypeScript/blob/master/lib/lib.dom.d.ts#L3719

**Comment by sgen** [comment]
Just ran into this when trying to call addEventListener from a wrapper function:

```typescript
export function name(event: string, listener: function, opts: Object|boolean): void {}
```

**Comment by HolgerJeromin** [comment]
This is not fixed for document and others in TS 2.4.1 (and ts playground)

````
document.addEventListener('readystatechange', function () { }, { once: true });
document.createElement('div').addEventListener('click', function () { }, { once: true });
````

**Comment by mhegazy** [comment]
@HolgerJeromin please file a new issue to track that.
